webapp/stores/error_store.jsx

```diff
@@ -70,7 +70,9 @@ class ErrorStoreClass extends EventEmitter {
 
         BrowserStore.removeGlobalItem('last_error');
         BrowserStore.removeGlobalItem('last_error_conn');
-        this.emitChange();
+        if (lastError) {
```

Comment:
Yes. This is just a guard so that we only bother to emit a clear message if there was actually an error to clear.